port/common/omrsysinfo.c

```diff
@@ -163,6 +163,40 @@ omrsysinfo_get_processor_feature_name(struct OMRPortLibrary *portLibrary, uint32
 	return rc;
 }
 
+/**
+ * Generate the corresponding string literals for the provided OMRProcessorDesc.
+ *
+ * @param[in] portLibrary The port library.
+ * @param[out] buff The cpu features output string.
+ * @param[in] BUFF_SIZE The size of the buff.
```

Comment:
You should mention that the buffer will be zero initialized and overwritten with the string contents. The size argument should also say "in number of bytes".